src/main/java/uk/gov/justice/digital/hmpps/csr/api/repository/NotificationRepository.kt

```diff
@@ -0,0 +1,14 @@
+package uk.gov.justice.digital.hmpps.csr.api.repository
+
+import org.springframework.data.jpa.repository.query.Procedure
+import org.springframework.data.repository.CrudRepository
+import org.springframework.stereotype.Repository
+import uk.gov.justice.digital.hmpps.csr.api.model.ShiftNotification
+import java.util.*
+
+
+@Repository
+interface NotificationRepository : CrudRepository<ShiftNotification, UUID>{
+    @Procedure
+    fun GET_MODIFIED_SHIFTS(): Collection<ShiftNotification>
+}
```

Comment:
not read the docs, but I imagine the proc name goes in the goes in like this @Procedure("GET_MODIFIED_SHIFTS") and not in the method name. but I'm ready to be surprised. 😁

Reply:
You boi baeldung's got this one:

https://www.baeldung.com/spring-data-jpa-stored-procedures